addons/mail/static/src/core/web_portal/message_patch.js

```diff
@@ -14,12 +15,13 @@ patch(Message.prototype, {
 
     /**
      * @override
-     * @param {HTMLElement} element
+     * @param {HTMLElement} bodyEl
      */
-    prepareMessageBody(element) {
-        const $el = $(element);
-        $el.find(".o-mail-read-more-less").remove();
-        this.insertReadMoreLess($el);
+    prepareMessageBody(bodyEl) {
+        Array.prototype.map.call(bodyEl.querySelectorAll(".o-mail-read-more-less"), (el) =>
```

Comment:
```suggestion
        Array.from(bodyEl.querySelectorAll(".o-mail-read-more-less")).map((el) =>
```

I'm not confortable with this prototype syntax, I think it would be cleaner to use `Array.from`

Reply:
Actually `forEach` would be even better.